Source/WebKit/NetworkProcess/cache/NetworkCacheDataCurl.cpp

```diff
@@ -113,11 +113,11 @@ RefPtr<SharedMemory> Data::tryCreateSharedMemory() const
     if (isNull() || !isMap())
         return nullptr;
 
-    auto newHandle = std::get<FileSystem::MappedFileData>(*m_buffer).fileMapping();
-    if (!newHandle)
+    auto handle = std::get<FileSystem::MappedFileData>(*m_buffer).fileMapping();
+    if (!handle)
```

Comment:
You renamed this variable. But, I think old name is better because it indicated a new handle.